platform-sdk/platform-apps/tests/PlatformTestingTool/build.gradle.kts

```diff
@@ -15,31 +15,39 @@
  */
 
 plugins {
-  id("com.swirlds.platform.conventions")
-  id("com.swirlds.platform.application")
-  id("com.swirlds.platform.protobuf")
+    id("com.swirlds.platform.conventions")
+    id("com.swirlds.platform.application")
+    id("com.google.protobuf")
```

Comment:
We should make sure this is correct.  We replaced google's protobuf compiler in the main codebase, so I would think the testing tool should use the same plugin.

Reply:
Right now, this is preserving what is on `develop`. Just moved things around.

Maybe we can address this concern in a follow up once this is merged?

Reply:
It is currently `com.swirlds.platform.protobuf`, and this PR changes to `com.google.protobuf`.  That's different from develop, so I'm asking if that's correct, given we don't directly use google's protobuf implementation with the switch to PBJ.

Reply:
@jsync-swirlds PTT has never used PBJ and has always used the official google protobuf implementation and compiler. @jjohannes changes here are correct. The removal of the gradle `id("com.swirlds.platform.protobuf")` custom plugin is due to consolidating the changes into the `build.gradle.kts` definition of the project.